server/Repository/UserRepository.cs

```diff
@@ -1,9 +1,25 @@
 
 
+using Microsoft.EntityFrameworkCore;
+using server.Context;
+using server.Models.DTOs;
+
 namespace server.Repository
 {
-    public class UserRepository
+    public class UserRepository : IUserRepository
     {
-        
+        private readonly OfficeDbContext dbContext;
+
+        public UserRepository(OfficeDbContext dbContext)
+        {
+            this.dbContext = dbContext;
+        }
+
+        public Task<List<UserDto>> GetAllUsers()
+        {
+            return dbContext.Users.Select(user =>
+                new UserDto(user.Id, user.Name, user.Email, null)
```

Comment:
Hvorfor er det null for booking her? hvordan får vi se detaljer om en persons booking historikk fra Apis?

Reply:
Satte denne til null, da en User ikke behøver å ha bookings. Men er enig med deg, det gir jo ikke mening hvis jeg mapper over til bookings å ikke ta med bookings :D :p 

Endrer koden til å mappe over med Mappers i Helper package.